tests/Unit/GitlabActionsCommandTest.php

```diff
@@ -41,6 +52,34 @@ public function testExecute() {
       ->assertOutputContains('"project_id": 12345');
   }
 
+  public function testMissingVars() {
+    $required_variables = [
+      'GITLAB_PRIVATE_TOKEN',
+      'CI_SERVER_URL',
+    ];
+    $client = $this->mockClient(FALSE);
+    foreach ($required_variables as $variable) {
+      putenv("$variable=");
+      TestCommand::for(new GitlabActionsCommand($client->reveal()))
+        ->execute('mergeRequests.all')
+        ->assertOutputContains("\"$variable\" env variable is not set.")
+        ->assertStatusCode(1);
+      putenv("$variable=something");
+    }
+  }
+
+  public function test() {
```

Comment:
Can we specialize a bit more the test method name?

Reply:
fixed